.github/workflows/causal_lm_cpp.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches:
       - master
       - 'releases/**'
+    paths-ignore:
+      - .github/workflows/llm_bench-python.yml
+      - tools/**
```

Comment:
looks like it will not work, because last job within this pipeline `ci/gha_overall_status_linux` is set as required in github repo settings. So, your PR will just stuck on CI run.

Reply:
Ok. Shall I remove `paths-ignore` section for now?

Reply:
yes :(